### ISSUE TITLE
Initial values on subclass overwritten by parent class

### ISSUE THREAD

**Issue by TonyFlury** [TonyFlury/py-importjson#8]
Due to ordering of initialisation, any attributes defined on the subclass are overwritten by attributes on the parent class - this includes any constraints.

This is not intended behaviour.

Json class definitions : 

``` json
    {
        "classA": {  "__doc__":"Class A", "x":1, "y":2  },
        "classB":{"__doc__":"Class B", "__parent__":"classA", "x":3, "y":4 }
    }
```

After Import : 

Expected 

``` python
    instB = classB()
    instB.x == 3
    instB.y == 4
```

Actual

``` python
    instB = classB()
    instB.x == 1
    instB.y == 2
```

i.e. the Attributes defined for ClassB have not overridden those  defined for Class A

**Comment by TonyFlury** [comment]
See wiki page [Issue #3 Proposals](https://github.com/TonyFlury/py-importjson/wiki/Issue-%233---Proposals) for further analysis of Issue 3 and this Issue.

**Comment by TonyFlury** [comment]
Will be resolved as follows : 
All classes will consistently call super when they have a parent class.
super will be called, and then the initial values will be set - the values will be literal, i.e. attributes wont be set from values in the `_constraints` attributes.

`test_050_001_ClassInheritanceOverrideAttr` in the module test set tests that the initial value of child class attributes override parent class attributes of the same name. This test must pass when this issue is solved.

**Comment by TonyFlury** [comment]
Will be resolved by calling super first, and then setting the instance attributes.

**Comment by TonyFlury** [comment]
Fixed in 0.0.1a8
